src/Python/Core/py3d_registration.cpp

```diff
@@ -35,6 +35,12 @@
 #include <Core/Registration/FastGlobalRegistration.h>
 #include <Core/Registration/ColoredICP.h>
 
+// #include <Core/Geometry/Image.h>
+#include <Core/Geometry/RGBDImage.h>
+#include <Core/Geometry/TriangleMesh.h>
+#include <Core/Registration/ColorMapOptimization.h>
+#include <Core/Camera/PinholeCameraTrajectory.h>
```

Comment:
Can we place ColorMapOptimization under py3d_registration? Can add these headers? Maybe this would not match the category.